content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -109,13 +110,101 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
+  IngressClass. Implementations can treat this as a separate PathType or treat
+  it identically to Prefix or Exact path types
```

Comment:
nit:
```suggestion
  it identically to Prefix or Exact path types.
```

Reply:
Alternatively, if using the revised formatting:
```suggestion
  it identically to `Prefix` or `Exact` path types.
```

Reply:
Updated, thanks!